### PR TITLE
Server routes and readme

### PR DESCRIPTION
**Pull request by ucb15erickao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,67 @@
...
-
 
 ## Table of Contents
-1.[Usage](#Usage)
+1. [Usage](#Usage)
+2. [API](#API)
 
 
-##Usage
+## Usage
```
Comment (ryanminjaeshin) [inline README.md:10]: I should add this to my README too. Good job!

File: README.md [hunk 2]
```diff
@@ -1,15 +1,67 @@
...
 2. Run 'npm run react-dev' to transpile.
 3. Run 'npm run start-server' and navigate to localhost:1234 in the browser.
+
+
+## API
+**GET:**\
+Request: pid (number)\
+Response: array of recommended products
+```
+[{
```
Comment (ryanminjaeshin) [inline README.md:21]: I also need to put my products in the array. I gotta change mine lol

File: server/index.js [hunk 3]
```diff
@@ -17,6 +17,22 @@ app.get('/:pid/recommendation/getInfo', (req, res) => {
   });
 });
 
+app.get('/:pid', (req, res) => {
```
Comment (ryanminjaeshin) [inline server/index.js:20]: Are you planning to edit the endpoint later on? For example, product/:pid ?

**Review by ryanminjaeshin — APPROVED** [review]
Looks good!

File: server/index.js (hunk 3 above)
Comment (ucb15erickao) [reply server/index.js:20]: Maybe I should, that would look more realistic.

**Pull request merged**